cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -18,16 +18,16 @@ define(
                 this.postUrl = '/test/post/url';
                 this.uploadButton = $('<button>');
                 this.videoSupportedFileFormats = ['.mp4', '.mov'];
+                this.videoUploadMaxFileSizeInGB = 5;
                 this.view = new ActiveVideoUploadListView({
                     concurrentUploadLimit: concurrentUploadLimit,
                     postUrl: this.postUrl,
                     uploadButton: this.uploadButton,
-                    videoSupportedFileFormats: this.videoSupportedFileFormats
+                    videoSupportedFileFormats: this.videoSupportedFileFormats,
+                    videoUploadMaxFileSizeInGB: this.videoUploadMaxFileSizeInGB
                 });
                 this.view.render();
                 jasmine.Ajax.install();
-                this.globalAjaxError = jasmine.createSpy();
```

Comment:
We removed this because we are now not showing globalAjaxError that is default studio erorr handler. We have implemented our own version of the error notification on the file failure method.